_posts/2024-03-03.quarkus-redhat-strategy.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: 'Quarkus & Red Hat''s evolving middleware strategy'
+date: 2025-03-03
+tags: announcement
+synopsis: 'Information about the recent announcement and the future of Quarkus'
+author: maxandersen
+---
+
+Over the past weeks, we've received questions about the https://www.redhat.com/en/blog/evolving-our-middleware-strategy[recent announcement] that Red Hat's Middleware engineering and product teams, which includes Quarkus, are moving to IBM.
+
+We want to take this to acknowledge this change while reaffirming what has always been true:
+
+*Quarkus is an open-source project driven by a global community of contributors, spanning companies, individuals, and open-source enthusiasts.*
+
+From its inception, Quarkus has been shaped by *real-world adoption* across industries, with companies https://quarkus.io/userstories/[sharing their user-stories], leveraging Quarkus's unmatched performance and developer joy in the Java ecosystem. We recently celebrated our https://quarkus.io/1000contributors/[*1000th contributor milestone*], a testament to the strength of this diverse and thriving community.
+
+IBM and Red Hat have clearly communicated that the current and future contributions to Quarkus are a key component of the middleware strategy. IBM has already been an active contributor to Quarkus, and we look forward to increased collaboration - alongside the many other individuals and companies who contribute, innovate, and drive Quarkus forward.
+
+Additionally, *Quarkus is already on the path to becoming part of https://www.commonhaus.org/[Commonhaus] an open-source foundation*, further ensuring its long-term independence and success as a truly community-driven project.
```

Comment:
I’d add. 
(Yeah the timing looks suspicious but the foundation move decision is unrelated, if anything it reaffirms the power of a foundation hosted project _add link to blog about our reasons_)

Reply:
This blog is avoid shining light on the elephant in the room. I think the foundation move is part of that elephant so lets make it clear but concise )